WordPress/wordpress.org
# Add a new tab to merge the "History", "Other locales" and "Translation Memory" tabs

## Problem



Currently, we have 5 tabs in the right sidebar to show different information:
- Meta.
- Discuss.
- History.
- Other locales.
- Translation memory (TM).

![image](https://github.com/WordPress/wordpress.org/assets/1667814/490dd241-cb6e-4f3b-ae08-1ef993afcf46)

These tabs take up a lot of horizontal space with small devices and/or screens, showing tabs with only a few letters and without the number of elements, with a bad usability.

![image](https://github.com/GlotPress/gp-translation-helpers/assets/1667814/b2912bb1-e4e0-4961-a8ec-5b442e2097ef)

## Solution



This PR merges the content from the "History", "Other locales"  and "TM" in a new tab, called "Others", removing the three previous tabs.

![image](https://github.com/WordPress/wordpress.org/assets/1667814/cfbfe930-5748-4c02-a8c1-e7ce69b7867e)


This PR adds "summary" elements, so the user could show or hide the different elements. This PR remembers the status of each summary (opened or closed) when you change to another row.

![image](https://github.com/WordPress/wordpress.org/assets/1667814/f7a82423-47e6-4a29-8821-8471c4294716)

This PR also solves a problem with the "Other locales" UI, reported [here](https://meta.trac.wordpress.org/ticket/7124).

**Problem**

![image](https://github.com/WordPress/wordpress.org/assets/1667814/9b6d78cf-e2ca-4fce-aead-fd7186526a8e)

**Solution** (singular): [Example](https://translate.wordpress.org/projects/wp/6.1.x/ar/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=15420267&filters%5Btranslation_id%5D=101428768)

![image](https://github.com/WordPress/wordpress.org/assets/1667814/171673e0-30c1-431e-bfbc-639ee67a400b)

**Solution** (plural): [Example](https://translate.wordpress.org/projects/wp/6.1.x/ar/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=15418382&filters%5Btranslation_id%5D=101426762)

![image](https://github.com/WordPress/wordpress.org/assets/1667814/70e896ca-0851-4c72-ab3e-3c9dd5de4d35)

And it resolves and overflow problem in the "History" table:

**Problem**

![image](https://github.com/WordPress/wordpress.org/assets/1667814/4de082c7-2dfe-4eb8-a7d5-d1c286544ace)

**Solution**

![image](https://github.com/WordPress/wordpress.org/assets/1667814/2afbc977-c32a-4433-b7a9-962b420ba7ab)

## Testing instructions



To test this PR, you need to use the branch from this [gp-translation-helpers' PR](https://github.com/GlotPress/gp-translation-helpers/pull/179). 

To test this PR, you have to access to some projects with:
- 0 or some elements in the "Translation memory" summary.
- 0 or some elements in the "Other locales" summary. 
- 0 or some History summary. 

You need to check some of these combinations to be sure all work as expected.
